pkg/validate/validate.go

```diff
@@ -21,6 +23,28 @@ var (
 	}()
 )
 
+// AdditionalTrustBundle checks if the given string contains valid certificate(s) and returns an error if not.
+func AdditionalTrustBundle(v string) error {
```

Comment:
this should probably be `CABundle` as that's what's is validating ie that the string is ca bundle.